crates/booster/src/lib.rs

```diff
@@ -34,6 +38,138 @@ impl LowState {
     }
 }
 
+impl LowState {
+    pub fn joint_positions(&self) -> Joints {
```

Comment:
implement from iterator for Joints, then this function can collapse to just. Same for the one below.
```rs
self.motor_state_serial.iter().map(|joint| joint.position).collect()
```